### ISSUE TITLE
Please allow warnings to be ignored

### ISSUE THREAD

**Issue by rrthomas** [wcandillon/xqlint#96]
For example, I'm using this apparently natural XQuery function from the [O'Reilly book](http://www.xqueryfunctions.com/xq/functx_repeat-string.html):

```xquery
declare namespace functx = "http://www.functx.com";
declare function functx:repeat-string
  ( $stringToRepeat as xs:string? ,
    $count as xs:integer )  as xs:string {

   string-join((for $i in 1 to $count return $stringToRepeat),
                        '')
 } ;
```

xqlint correctly complains that `$i` is unused. I don't see a natural way to rewrite the function without this variable (in particular, the variable is mandatory in a `for` clause), so it would be nice to be able to tell xqlint to ignore this error.

Thanks for xqlint!

**Comment by wcandillon** [comment]
That's a good point, the way this is solve in ESLint is to prefix the var name with `_`. I haven't worked on XQLint in many years but it might be something someone could easily contribute to the project.

**Comment by wcandillon** [comment]
Thanks for the PR :) it has been published in `0.5.1`

**Comment by rrthomas** [comment]
Thanks!
